reach

```diff
@@ -115,10 +115,143 @@ run_s () {
   sh -ac "$TMP/out.sh" "$0"
 }
 
+gen_reachc_id () {
+  REACH_ENV_DIR="${XDG_CONFIG_HOME:-$HOME/.config}/reach/"
```

Comment:
I don't think you need this because there's already `$CNF`.

Reply:
I did not realize this variable when I was writing this piece.

I have removed the REACH_ENV_DIR variable and used $CNF on REACH_ENV_FILE.
Thanks for flagging this.